changelogs/fragments/1589-return_false_when_no_change..yml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+- cloudwatchevent_rule - add scrub none inside the _targets_to_put method (https://github.com/ansible-collections/amazon.aws/pull/1589)."
```

Comment:
The changelog should describe what the end user needs to know, functionaly, about the change, rather than the implementation details of the code.  Something like 
"Fixes changed status to report False when no change has been made. The module had incorrectly always reported a change."  
But make sure it describes whatever scenario the wrong change status would have occurred for, if it wasn't always.